app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/IdempotentMigrations.java

```diff
@@ -35,12 +36,11 @@ private void migrateBGAlerts() {
                 highMark = highMark * Constants.MMOLL_TO_MGDL;
                 lowMark = lowMark * Constants.MMOLL_TO_MGDL;
             }
-            int bg_snooze = Integer.parseInt(prefs.getString("bg_snooze", "20"));
             boolean bg_sound_in_silent = prefs.getBoolean("bg_sound_in_silent", false);
             String bg_notification_sound = prefs.getString("bg_notification_sound", "content://settings/system/notification_sound");
 
-            AlertType.add_alert(null, "High Alert", true, highMark, true, 1, bg_notification_sound, 0, 0, bg_sound_in_silent, bg_snooze);
-            AlertType.add_alert(null, "Low Alert", false, lowMark, true, 1, bg_notification_sound, 0, 0, bg_sound_in_silent, bg_snooze);
+            AlertType.add_alert(null, "High Alert", true, highMark, true, 1, bg_notification_sound, 0, 0, bg_sound_in_silent, SnoozeActivity.getDefaultSnooze(true));
```

Comment:
Im kind of against this change, if someone previously set their snooze (with the old style alerts) I think we should keep that value. Is there any reason for this change otherwise?